src/pydata_sphinx_theme/__init__.py

```diff
@@ -25,41 +25,6 @@ def update_config(app):
     theme_options = utils.get_theme_options_dict(app)
     warning = partial(utils.maybe_warn, app)
 
-    # TODO: deprecation; remove after 0.14 release
```

Comment:
Can you confirm this was already removed long ago ? I didn't find it in theme.conf

Reply:
- marked as "deprecate after 0.11" (2022 Jun): https://github.com/pydata/pydata-sphinx-theme/pull/698/files#diff-f0f04e8b873a8565b32c276f0dfcfbdb49b0f8b1c022d8cb59cd179872f542dfR42-R43
- Removed from `theme.conf` (2023 Feb): https://github.com/pydata/pydata-sphinx-theme/commit/9a10f9d578739f20ea9c30a6c62cff1ead5c28f6#diff-f0f04e8b873a8565b32c276f0dfcfbdb49b0f8b1c022d8cb59cd179872f542dfL52-L53